price-feeder/oracle/oracle_test.go

```diff
@@ -200,18 +209,46 @@ func (ots *OracleTestSuite) TestPrices() {
 				},
 			},
 		},
+		config.ProviderHuobi: mockProvider{
+			prices: map[string]provider.TickerPrice{
+				"USDCUSD": {
+					Price:  sdk.MustNewDecFromStr("1"),
+					Volume: sdk.MustNewDecFromStr("2396974.34000000"),
+				},
+			},
+		},
+		config.ProviderCoinbase: mockProvider{
+			prices: map[string]provider.TickerPrice{
+				"USDTUSD": {
+					Price:  sdk.MustNewDecFromStr("1"),
+					Volume: sdk.MustNewDecFromStr("1994674.34000000"),
+				},
+			},
+		},
+		config.ProviderOsmosis: mockProvider{
+			prices: map[string]provider.TickerPrice{
+				"XBTUSDT": {
+					Price:  sdk.MustNewDecFromStr("3.717"),
+					Volume: sdk.MustNewDecFromStr("1994674.34000000"),
+				},
+			},
+		},
```

Comment:
Please note that now the price feeder will calculate and store prices for assets which are not on the oracle module's accept list, because now we may be using them for conversion rates